src/OnlineSales/Entities/UploadedImage.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="UploadedImage.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+
+namespace OnlineSales.Entities
+{
+    public class UploadedImage : BaseEntity
+    {
+        [Required]
+        public string ScopeId { get; set; } = string.Empty;
+
+        public string FileName { get; set; } = string.Empty;
```

Comment:
FileName -> Name - no need to say it is a file, we know that every image is a file, so you can just say name, extension, etc.